src/v2/view-builder/utils/LinksUtil.js

```diff
@@ -99,7 +99,7 @@ const getSkipSetupLink = (appState) => {
         'type': 'link',
         'label': loc('oie.enroll.skip.setup', 'login'),
         'name': 'skip-setup',
-        'formName': RemediationForms.SKIP,
+        'actionPath': RemediationForms.SKIP,
```

Comment:
Why is this not a `formName`? From the response I can see that it is inside the remediation object...

I can see that we don't want this to submit a form, but instead just follow the link. So I am guessing that is the reason, which makes sense. With that said, then I think we should not have `RemediationForms` constants assigned to an `actionPath`, those should probably be used only for `formNames`
I see we already have a mix in our codebase though... so as a nit, maybe we change it to:
```
'actionPath': 'skip',
```
which is what we already have in [ReEnrollAuthenticatorWarningPasswordView](https://github.com/okta/okta-signin-widget/blob/master/src/v2/view-builder/views/password/ReEnrollAuthenticatorWarningPasswordView.js#L46)

Reply:
Yea i was thinking about this too.. I thought the action path needs to be the form name (https://github.com/okta/okta-signin-widget/blob/master/src/v2/controllers/FormController.js#L113), so I set it to this constant. Will change 👍

Reply:
Talked to Mau, leaving as is for now.. Will discuss with the team later

Reply:
It shall be `actionPath`.
- `actionPath` :  find the action from remediation list or other top level object and submit request
- `formName` : re-render page using this *remedation* form.